docs/getting-started.md

```diff
@@ -52,10 +52,12 @@ Remove nodes
 You may want to remove **worker** nodes to your existing cluster. This can be done by re-running the `remove-node.yml` playbook. First, all nodes will be drained, then stop some kubernetes services and delete some certificates, and finally execute the kubectl command to delete these nodes. This can be combined with the add node function, This is generally helpful when doing something like autoscaling your clusters. Of course if a node is not working, you can remove the node and install it again.
 
 - Add worker nodes to the list under kube-node if you want to delete them (or utilize a [dynamic inventory](https://docs.ansible.com/ansible/intro_dynamic_inventory.html)).
+- Use `--extra-vars "node=<nodenname>"` to select the node you want to delete
```

Comment:
node=node1,node2,node3,......    Is that like this？ when I delete nodes.

Reply:
I'am not sure if this will work in the pre/post tasks.

For `hosts:` it is ok, but here 
```with_items:
-    - "{{ node }}"
```
i am not sure if ansible will make an array from a `,` seperated list. 
But i will test it.

Reply:
Ok added it